sourcegraph/jetbrains
# Manage multiple chat conversations

## Test Plan

1. Start a new chat
2. Send message similar to `my favorite color is blue`
3. Close IDE
4. Run IDE and open previous conversation
5. Ask `what's my favorite color?`
6. Response should be similar to `your favorite color is blue`

This is happy path. More optional ideas for checking corner cases are added to `TESTING.md` in this PR.

XML with serialized chats is located in `build/idea-sandbox/config/options/cody_history.xml`.
